src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs

```diff
@@ -61,7 +35,6 @@ internal AsymmetricAdapter(SecurityKey key, string algorithm, bool requirePrivat
         {
         }
 
-        // This constructor will be used by NET45 for signing and for RSAKeyWrap
         internal AsymmetricAdapter(SecurityKey key, string algorithm, HashAlgorithm hashAlgorithm, bool requirePrivateKey)
```

Comment:
Do we need the contructor at all?

Reply:
Yes, the constructor above it refers to it, and that one is used.